pom.xml

```diff
@@ -2648,10 +2760,73 @@
     <profile>
       <id>hadoop-3.1</id>
```

Comment:
@HyukjinKwon @srowen
I'd like to upgrade hadoop-3.1 to hadoop-3.2 to workaround [HADOOP-16086](https://issues.apache.org/jira/browse/HADOOP-16086). Otherwise the testing can not fully passed.

Reply:
That seems fine. You could just call the profile hadoop-3 while at it, until we have a reason to differentiate different versions, which I hope we never do.

Reply:
Yup, I'm fine with it. Alternatively you can also skip some tests with the hadoop version conditions if the test failures are not a lot, and then separately do that later.